### PR TITLE
#244: Re-enable UpdateShippedCodeTest

### PR DESCRIPTION
**Pull request by mor-n4**
see #244

### CODE REVIEW

**Comment by mor-n4** [comment]
builds green

File: plugins/org.eclipse.n4js.external.libraries/shipped-code/runtime/n4js-node/node_modules/encoding/package.json [hunk 1]
```diff
@@ -17,9 +17,6 @@
   "license" : "MIT",
```
Comment (qtran-n4) [inline plugins/org.eclipse.n4js.external.libraries/shipped-code/runtime/n4js-node/node_modules/encoding/package.json]: Just for my understanding: Are these package.json generated?

Comment (mor-n4) [reply plugins/org.eclipse.n4js.external.libraries/shipped-code/runtime/n4js-node/node_modules/encoding/package.json]: not generated; they are part of the npm packages used by "n4js-node"

**Comment by mor-n4** [comment]
I changed the `npm-shrinkwrap.json` to `npm-lock.json` manually (but they are also modified by `npm install`). Also added `.npmrc` manually.

**Comment by mor-n4** [comment]
thanks! merging now ...

**Pull request merged**